cmd/admxgen/defs/mount.yaml

```diff
@@ -26,3 +26,32 @@
   type: "mount"
   meta:
     strategy: "append"
+
+- key: "/system-mounts"
+  displayname: "System mounts"
+  explaintext: |
+    Define network shares that will be mounted for the system.
+    If more shares are defined higher in the GPO hierarchy, the entries listed here will be appended to the list and duplicates will be removed.
+    
+    Values should be in the format: 
+        <protocol>://<hostname-or-ip>/<shared-dir>
+    e.g.
+        nfs://example_nfs.com/nfs_shared_dir
+        cifs://example_cifs.com/cifs_shared_dir
+        fuse://ftp_share_server.com
```

Comment:
We accept [both](https://github.com/ubuntu/adsys/blob/f6cf6e2d06391a646651201533b2dc5dd836ba84/internal/policies/mount/mount.go#L310) 😄.

However, as I've said in the commit rationale, we should list the correct input in the policy to avoid creating some "laziness"

Reply:
For the user is different... Gio supports the smb:// directly, so there's no need to do it.

Reply:
After what we talked in IIRC, about gio having to use smb:// instead of cifs:// and so on, do you think we can merge this?